support-frontend/assets/components/cardClickable/cardClickable.tsx

```diff
@@ -0,0 +1,188 @@
+import { css } from '@emotion/react';
+import {
+	brand,
+	focus,
+	from,
+	headline,
+	neutral,
+	space,
+	textSans,
+	until,
+} from '@guardian/source-foundations';
+import { SvgChevronRightSingle } from '@guardian/source-react-components';
+import { useEffect } from 'react';
+
+const containerCss = css`
+	:not(:last-child) {
+		margin-bottom: ${space[2]}px;
+
+		${from.tablet} {
+			margin-bottom: ${space[4]}px;
+		}
+	}
+`;
+
+const cardCss = css`
+	padding: ${space[3]}px ${space[3]}px ${space[4]}px;
+	background-color: ${neutral[100]};
+	color: ${neutral[7]};
+	border: 1px solid ${neutral[86]};
+	border-radius: ${space[3]}px;
+
+	&:hover {
+		cursor: pointer;
+	}
+
+	/* .src-focus-disabled is added by the Source FocusStyleManager */
+	html:not(.src-focus-disabled) &:focus {
+		outline: 5px solid ${focus[400]};
+		outline-offset: -5px;
+	}
+
+	${from.desktop} {
+		padding: ${space[5]}px ${space[6]}px;
+	}
+`;
+const topCss = css`
+	${from.desktop} {
+		margin-left: 2px;
+	}
+`;
+const bottomCss = css`
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+
+	${until.desktop} {
+		margin-left: ${space[1]}px;
+	}
+`;
+const headingCss = (color: string) => css`
+	${headline.xxsmall({ fontWeight: 'bold', lineHeight: 'regular' })};
+	color: ${color};
+
+	${from.desktop} {
+		${headline.xsmall({ fontWeight: 'bold', lineHeight: 'tight' })};
+	}
+`;
+const paraCss = css`
+	text-align: left;
+	${textSans.small({ lineHeight: 'regular' })};
+
+	${from.desktop} {
+		${textSans.medium({ lineHeight: 'regular' })};
+	}
+`;
+const chevronCss = css`
+	color: #606060;
+	background-color: transparent;
+	&:hover {
+		background-color: transparent;
+	}
+	& svg {
+		width: ${space[6]}px;
+		margin: 2px;
+	}
+`;
+
+export type CardClickableProps = {
+	cardTitle: string;
+	cardParagraph: string;
+	onCardClick: () => void;
+};
+
+export function CardClickable({
+	cardTitle,
+	cardParagraph,
+	onCardClick,
+}: CardClickableProps): JSX.Element | null {
+	const cardClickableProp = { onCardClick };
+
+	useEffect(() => {
+		const cardElement = document.querySelector('.card');
```

Comment:
I really hate to say this because I can tell this has been a lot of work, but I think you have seriously over-engineered this. The moment you find yourself using browser selection APIs inside a React component, such as `document.querySelector`, it should give you serious pause because you're stepping outside React's rendering behaviour. If you absolutely _have_ to access a DOM node directly in React code you should use [refs](https://react.dev/learn/manipulating-the-dom-with-refs), but this should be avoided if at all possible. The point of using React is that we don't have to think about rooting around in the DOM manually- we use JSX to describe the DOM that we want to create, and React handles actually rendering it and keeping it up to date.

I would honestly favour a pure CSS approach- using an actuall `<a>` tag with succinct but descriptive text, (in this case probably using the title) and applying an `::after` pseudo-element to it that covers the whole 'card' component. This is the approach used on dotcom fronts and gov.uk. It does mean the text in the card is not selectable (which is why I just strongly dislike the 'clickable card' as a component type in the first place), but it's _much_ less code to document and maintain and means we are actually using proper semantic HTML rather than attaching event listeners to elements like `<div>`s that have no semantic meaning.

When it comes to web accessibility, proper semantic HTML should always be the first thing you reach for, and anything extra that we need to add with ARIA attributes, extra JS, etc should be kept to the absolute minimum that we can feasibly get away with. Browsers are already built to communicate a page's structure to screenreaders and other kinds of assistive technology, and we should strive to rely as much as possible on what the browser can do for us for free, rather than trying to create DIY solutions that we then have to maintain ourselves.